rest/model/subscriber.go

```diff
@@ -163,6 +192,40 @@ func (s *APIGithubPRSubscriber) ToService() (interface{}, error) {
 	}, nil
 }
 
+func (s *APIGithubMergeSubscriber) BuildFromService(h interface{}) error {
+	if v, ok := h.(event.GithubMergeSubscriber); ok {
```

Comment:
I think this is redundant with the switch statement below. Each is a different way of checking the type.

Reply:
This makes sure we are dealing with a particular type (the _pointer_ to GithubMergeSubscriber in this case). 
This way we can copy information out of it knowing the type of the interface. Were we not to do this, the type would still be ambiguous (it might be an actual value)

Reply:
We could do it that way but would need to duplicate the case statement:
 	case *event.GithubMergeSubscriber:
 		s.Owner = ToAPIString(v.Owner)
 		s.Repo = ToAPIString(v.Repo)
                 ...
 	case event.GithubMergeSubscriber:
 		s.Owner = ToAPIString(v.Owner)
 		s.Repo = ToAPIString(v.Repo)
                 ...

It won't like if we do this:
 	case *event.GithubMergeSubscriber, event.GithubMergeSubscriber :
 		s.Owner = ToAPIString(v.Owner)
 		s.Repo = ToAPIString(v.Repo)
                 ...
since the type is ambiguous.

Reply:
should be good now (per our discussion)